### ISSUE TITLE
specplots not updating for USAXS

### ISSUE THREAD

**Issue by prjemian** [prjemian/spec2nexus#129]
parser fails on file with multiple `#F` sections

```
#= 18330 --faulthandler---------- module enabled
[specplotsAllScans.py 18330 2018-11-12 10:40:02.293394] faulthandler: module enabled
Traceback (most recent call last):
  File "/home/beams/USAXS/Documents/eclipse/USAXS/livedata/specplotsAllScans.py", line 344, in <module>
    plotAllSpecFileScans(specFile)
  File "/home/beams/USAXS/Documents/eclipse/USAXS/livedata/specplotsAllScans.py", line 127, in plotAllSpecFileScans
    for scan_number in sd.getScanNumbers():
  File "/APSshare/anaconda/x86_64/lib/python2.7/site-packages/spec2nexus/spec.py", line 294, in getScanNumbers
    return sorted(self.scans.keys(), key=int)
ValueError: invalid literal for int() with base 10: '35.1'
#= 18317 --Done---------------------------------- Mon Nov 12 10:40:02 CST 2018
```

Also: Exception *should* also report the file name (absolute, full path)

From [APS USAXS](https://github.com/APS-USAXS/ipython-usaxs/issues/76)

**Comment by prjemian** [comment]
Look at this fix: https://github.com/APS-USAXS/livedata/issues/3

**Comment by prjemian** [comment]
Is this resolved now?

**Comment by prjemian** [comment]
The exception was resolved with #128, still need to report the file name with the exception
